### DISCUSSION TITLE
Query regarding scalability

### DISCUSSION THREAD

**Discussion by geeky-programmer-exe** [hasura/graphql-engine#6939] [General]
This issue might seem trivial but how many CRUD requests can self hosted hasura handle?

**Comment by rikinsk** [comment]
It depends on a lot of factors. From the top of my head, the resources (memory/cpu) available to your Hasura instance, the resources available to your database, the connection settings between Hasura and the database, the amount of data in your database, the complexity of your individual CRUD requests, and so on.

Such questions are best answered by running benchmarks with the kind of requests and setup you would be working with as the answer could vary massively based on your exact use case
